miasm/core/parse_asm.py

```diff
@@ -174,7 +175,7 @@ def parse_txt(mnemo, attrib, txt, loc_db):
         # label
         match_re = LABEL_RE.match(line)
         if match_re:
-            label_name = match_re.group(1).encode()
+            label_name = force_str(match_re.group(1))
```

Comment:
```suggestion
            label_name = match_re.group(1)
```
Unless I am missing something here, since `LABEL_RE` is a `Pattern[str]`, `match_re` is a `Match[str]` and `match_re.group(1)` is a `str`.

Reply:
Correct @p-l- !
Thank you for the feed back. I will update the PR soon